book/06-github/sections/3-maintaining.asc

```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
 Ito ay nagpapahintulot sa iyo na suriin ang merge bago pindutin ang pindutan.
 
 
-===== Pull Requests on Pull Requests
+===== Mga Kahilingan na Pull sa mga Kahilingan na Pull
 
-Not only can you open Pull Requests that target the main or `master` branch, you can actually open a Pull Request targeting any branch in the network.
-In fact, you can even target another Pull Request.
+Hindi ka lamang magbubukas ng mga Kahilingan na Pull na naka-target sa prinsipal o branch na `master`, maaari ka talagang magbukas ng isang Kahilingan na Pull na nagta-target sa anumang branch sa network.
+Sa katunayan, maaari ka ring mag-target sa ibang Kahilingan na Pull.
 
-If you see a Pull Request that is moving in the right direction and you have an idea for a change that depends on it or you're not sure is a good idea, or you just don't have push access to the target branch, you can open a Pull Request directly to it.
+Kung nakikita mo ang isang Kahilingan na Pull na gumagalaw sa tamang direksyon at mayroon kang ideya para sa isang pagbabago na nakadepende dito o hindi ka sigurado na magandang ideya, o wala ka lamang access sa pag-push sa branch ng target, maaari kang magbukas ng isang Kahilingan na Pull na direkta dito.
 
-When you go to open a Pull Request, there is a box at the top of the page that specifies which branch you're requesting to pull to and which you're requesting to pull from.
-If you hit the ``Edit'' button at the right of that box you can change not only the branches but also which fork.
+Kapag ikaw ay nagbubukas ng isang Kahilingan na Pull, may isang kahon sa itaas ng pahina na tumutukoy kung aling branch ang hinihingan mo na maka-pull at kung saan hinihiling mo na mag-pull.
+Kung pindutin mo ang ``Edit'' na pindutan sa kanan ng kahon na iyon, maaari kang magbago hindi lamang sa mga branch kundi rin sa kung anong fork.
 
 [[_pr_targets]]
-.Manually change the Pull Request target fork and branch.
-image::images/maint-04-target.png[PR targets]
+.Baguhin nang mano-mano ang fork na target ng Kahilingan na Pull at branch.
```

Comment:
Baguhin nang mano-mano ang fork na target ng Kahilingan na Pull at branch.  to  Mano-manong baguhin ang Kahilingan ng Pull na target ang fork at branch